core/services/ocr3/plugins/ccip/internal/reader/home_chain_test.go

```diff
@@ -0,0 +1,210 @@
+package reader
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	mapset "github.com/deckarep/golang-set/v2"
+	libocrtypes "github.com/smartcontractkit/libocr/ragep2p/types"
+
+	"github.com/smartcontractkit/ccipocr3/internal/mocks"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+	"github.com/smartcontractkit/libocr/commontypes"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+var (
+	chainA       = cciptypes.ChainSelector(1)
+	chainB       = cciptypes.ChainSelector(2)
+	chainC       = cciptypes.ChainSelector(3)
+	oracleAId    = commontypes.OracleID(1)
+	p2pOracleAId = libocrtypes.PeerID{byte(oracleAId)}
+	oracleBId    = commontypes.OracleID(2)
+	p2pOracleBId = libocrtypes.PeerID{byte(oracleBId)}
+	oracleCId    = commontypes.OracleID(3)
+	p2pOracleCId = libocrtypes.PeerID{byte(oracleCId)}
+)
+
+func TestHomeChainConfigPoller_HealthReport(t *testing.T) {
+	homeChainReader := mocks.NewContractReaderMock()
+	homeChainReader.On(
+		"GetLatestValue",
+		mock.Anything,
+		"CCIPCapabilityConfiguration",
+		"getAllChainConfigs",
+		mock.Anything,
+		mock.Anything).Return(fmt.Errorf("error"))
+
+	configPoller := NewHomeChainConfigPoller(
+		homeChainReader,
+		logger.Test(t),
+		50*time.Millisecond,
+	)
+	_ = configPoller.Start(context.Background())
```

Comment:
Even if it cannot return an error now, it should check. (also, this function can return an error)
```suggestion
	err := configPoller.Start(context.Background())
	require.NoError(t, err)
```